tests/integration/helpers.py

```diff
@@ -40,3 +47,55 @@ def wait_for_condition(interval: int, timeout: int, condition: Callable):
 
         # Evil
         time.sleep(interval)
+
+
+def exec_test_command(args: List[str]):
+    process = subprocess.run(
+        args,
+        stdout=subprocess.PIPE,
+    )
+    assert process.returncode == 0
+    return process
+
+
+def exec_failing_test_command(args: List[str]):
+    process = subprocess.run(
+        args,
+        stderr=subprocess.PIPE,
+    )
+    assert process.returncode == 1
```

Comment:
There are some (rare) cases where we return a failed exit code other than `1`. Maybe this could become an optional parameter?

e.g.
```python
def exec_failing_test_command(args: List[str], expected_code: int = 1):
```